[IPE-370] Clarify MFA functionality when using Unified Identity account (#443)
hashicorp/terraform-docs-common: website/docs/cloud-docs/users-teams-organizations/2fa.mdx

Click **Settings** in your organization to reach your organization'a settings page, then click **Authentication**.

Click the button "Require two-factor". Please remember that all organization owners must have two-factor authentication on before this can be set.

## Requiring Two-factor Authentication for Users with HashiCorp Cloud Platform 

> When you require two-factor authentication for all users and have users who [log in with their HashiCorp Cloud Platform Account](/terraform/cloud-docs/users-teams-organizations/users#log-in-with-your-hashicorp-cloud-platform-account), the required configuration depends on the linked HashiCorp Cloud account identity:

Reviewer: ```suggestion
When you require two-factor authentication for all users and have users who [log in with their HashiCorp Cloud Platform Account](/terraform/cloud-docs/users-teams-organizations/users#log-in-with-your-hashicorp-cloud-platform-account), the required configuration depends on the linked HashiCorp Cloud account identity of the user who is trying to log in:
```

Reviewer: I'm still a little confused about this. Do we mean that admins have to determine all the login methods used in their org and configure 2FA separately for each login method? If so, where can they find the list of login methods?

Reviewer 2: this enforces that a user who needs access to the org must configure 2FA.  They are not allowed to access the org unless 2FA is configured.  Regarding HCP identity, the means of configuring 2FA differs depending on the identity chosen.

For HCP Email, they must configure HCP MFA
For HCP GitHub, they must configure GitHub MFA.

> When you require two-factor authentication for all users and have users who [log in with their HashiCorp Cloud Platform Account](/terraform/cloud-docs/users-teams-organizations/users#log-in-with-your-hashicorp-cloud-platform-account), the required configuration depends on the linked HashiCorp Cloud account identity:

Reviewer 2: maybe we can clarify this

"the required configuration for each Org member depends on their linked HashiCorp Cloud identity who is trying to access the Org."

> - **Email**: See [HashiCorp Cloud MFA](/hcp/docs/hcp/admin/iam/mfa).
> - **GitHub**: See [Configuring GitHub two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication).

Reviewer: ```suggestion
- **Email**: Follow the instructions in the [HashiCorp Cloud MFA](/hcp/docs/hcp/admin/iam/mfa) docs.
- **GitHub**: Follow the instructions in the [Configuring GitHub two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication) docs.
```

Reviewer: (Usually we try to avoid "see" for users who rely on screen readers.

> - **SSO**: currently HCP SSO accounts are not supported for TFC.  

Reviewer: ```suggestion
- **SSO**: Terraform Cloud does not currently support HCP SSO accounts.  
```
